src/hotspot/share/classfile/systemDictionaryShared.cpp

```diff
@@ -187,6 +187,11 @@ void SystemDictionaryShared::start_dumping() {
   _dump_in_progress = true;
 }
 
+void SystemDictionaryShared::stop_dumping() {
+  assert_lock_strong(DumpTimeTable_lock);
+  _dump_in_progress = true;
+}
+
```

Comment:
Did you really intend to set _dump_in_progress to true in stop_dumping()? start_dumping() also sets it to true.

Reply:
Oh, it should be `_dump_in_progress = false;`. I'll fix that.